talibs/ta/Makefile

```diff
@@ -1,6 +1,6 @@
 LDADD = -L$(link-out-dir)/../talib1 -ltalib1 \
-	-L$(link-out-dir)/../talib2 -ltalib2 \
-	-L$(link-out-dir)/../talib3 -ltalib3
+	-L$(link-out-dir)/../talib2 -ltalib2 #\
+#	-L$(link-out-dir)/../talib3 -ltalib3
```

Comment:
?

Reply:
Sorry, I did a `git commit -a` by mistake. Only the change in `talibs/host/main.c` was intended. I'll fix that in a new commit.